test/cljam/io/cram_test.clj

```diff
@@ -39,11 +39,16 @@
                (cram/read-refs cram-rdr)))
         (is (= (map fixup-bam-aln (sam/read-alignments bam-rdr))
                (cram/read-alignments cram-rdr)))))
-    (testing "read alignments in specified regions"
+    (testing "read alignments in specified regions (with and without index file)"
       (with-open [cram-rdr (cram/reader common/medium-cram-file
-                                        {:reference common/hg19-twobit-file})]
+                                        {:reference common/hg19-twobit-file})
+                  cram-rdr' (cram/reader common/medium-without-index-cram-file
+                                         {:reference common/hg19-twobit-file})]
         (is (cram/indexed? cram-rdr))
-        (are [?region ?count] (= ?count (count (cram/read-alignments cram-rdr ?region)))
+        (is (not (cram/indexed? cram-rdr')))
+        (are [?region ?count] (= ?count
+                                 (count (cram/read-alignments cram-rdr ?region))
+                                 (count (cram/read-alignments cram-rdr' ?region)))
```

Comment:
This test fails in my environment.
Maybe we need to reset the position of the mapped buffer before calling next `read-alignments`

Reply:
Thank you for pointing that out! Yeah, I'm just looking into it right now. I wasn't aware that I didn't run these tests on my local environment at all 😂

Reply:
b1b0ea4 should fix the position issue.
I also added 3623b37 to improve test coverage for the CRAM region read.